homeassistant/components/apprise/__init__.py

```diff
@@ -1 +1,55 @@
 """The apprise component."""
+
+import apprise
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_NAME, CONF_URL, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import config_validation as cv, discovery
+from homeassistant.helpers.typing import ConfigType
+
+from .const import CONF_FILE_URL, DATA_HASS_CONFIG, DEFAULT_NAME, DOMAIN
+
+type AppriseConfigEntry = ConfigEntry[apprise.AppriseConfig]
+
+PLATFORMS = [Platform.NOTIFY]
+
+CONFIG_SCHEMA = vol.Schema(
+    {
+        DOMAIN: vol.Schema(
+            {
+                vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+                vol.Optional(CONF_FILE_URL): cv.string,
+                vol.Optional(CONF_URL): cv.string,
+            }
+        )
+    },
+    extra=vol.ALLOW_EXTRA,
+)
```

Comment:
Okay so we need to keep using the already existing entrypoint, which is in notify.py. I do need to double check if there is a way to find out if the `get_service` is called via YAML or via `async_load_platform`